app/backend/src/couchers/servicers/search.py

```diff
@@ -458,6 +458,8 @@ def UserSearch(self, request, context):
 
             if request.only_with_references:
                 statement = statement.join(Reference, Reference.to_user_id == User.id)
+            if request.profile_completed:
+                statement = statement.where(User.has_completed_profile == True)
```

Comment:
I think this won't work since `profile_completed` is a `google.protobuf.BoolValue`. You'd have to do the following:

```py
            if request.HasField("profile_completed"):
                statement = statement.where(User.has_completed_profile == request.profile_completed.value)
```